docs/sources/auth/github.md

```diff
@@ -16,7 +16,7 @@ settings page). When you create the application you will need to specify
 a callback URL. Specify this as callback:
 
 ```bash
-http://<my_grafana_server_name_or_ip>:<grafana_server_port>/login/github
+http://<my_grafana_server_name_or_ip>:<grafana_server_port>/grafana/login/github
```

Comment:
Thank you @DanCech. I will open a new PR to get the content updated.

Reply:
@achatterjee-grafana looking around this is actually quite inconsistent between the different oauth setup pages, with one of the clearest being [GitLab](https://grafana.com/docs/grafana/latest/auth/gitlab/#create-gitlab-oauth-keys).  We should make this consistent across all the pages to avoid the kind of confusion that's led to this PR.